### ISSUE TITLE
new_after_n_chars not working in chunk_by_title function

### ISSUE THREAD

**Issue by Munz1993** [Unstructured-IO/unstructured#1640]
I am using partition_html function followed by chunk_by_title function as given below:

elements = partition(url='xyz', strategy="html", encoding='utf-8', skip_headers_and_footers = True)
chunks = chunk_by_title(elements, new_after_n_chars= 500) 

new_after_n_chars is supposed to limit the character length to 500. However, character length for some chunks is getting past 1500. I have also tried changing the default value of 1500 in function definition to 500, just to be sure. That's also giving the same results.

**Comment by cragwolfe** [comment]
Yes, `new_after_n_chars` is a "soft maximum" which does not divide an element to stay under that max. A new parameter `max_characters` was just introduced in https://github.com/Unstructured-IO/unstructured/pull/1540 for the "hard maximum" -- however at the moment it only applies to `Table` elements. This will soon change so that `max_characters` will be supported for all element types.
CC @amanda103 .
